CSS cookbook: update columns (#29961)
mdn/content: files/en-us/web/css/layout_cookbook/column_layouts/index.md

### A single row of items with equal heights — flexbox

> Flexbox can be used to break content into columns by setting {{cssxref("display", "display: flex;")}} to make a parent element a flex-container. Just adding this one property turns all the chidren (child elements, pseudo-elements, and text nodes) into flex items along a single line. Setting the same {{cssxref("flex")}} shorthand property with a single numeric distributes all the available space proportionally, generally making all the flex items the same size as long as none have non-wrapping content requires that item be larger.

Reviewer: ```suggestion
Flexbox can be used to break content into columns by setting {{cssxref("display", "display: flex;")}} to make a parent element a flex-container. Just adding this one property turns all the children (child elements, pseudo-elements, and text nodes) into flex items along a single line. Setting the same {{cssxref("flex")}} shorthand property with a single numeric value distributes all the available space equally, generally making all the flex items the same size as long as none have non-wrapping content forcing the item to be larger.
```

### Lining items up in rows and columns — grid layout

> If what you want a two-dimensional grid, a layout where items line up in rows and columns, then you should choose CSS Grid Layout. Similar to how flexbox works on the direct children of the flex container, grid layout works on the direct children of the grid container. Just set {{cssxref("display", "display: grid;")}} set on the container. Properties set on the grid container, like {{cssxref("grid-template-columns")}} and {{cssxref("grid-template-rows")}} define how the items are distribute both rows and columns.

Reviewer: ```suggestion
If you want a two-dimensional grid where items line up in rows _and_ columns, then you should choose CSS Grid Layout. Similar to how flexbox works on the direct children of the flex container, grid layout works on the direct children of the grid container. Just set {{cssxref("display", "display: grid;")}} on the container. Properties set on this container — like {{cssxref("grid-template-columns")}} and {{cssxref("grid-template-rows")}} — define how the items are distributed along rows and columns.
```
